Add named params & outer origin to diff operation

sculpt0r opened the pull request:
Add origin to the ckeditor4 repository which comes from CLI. Then diff target branch with a branch from that origin. Only if any slug for an external repository is passed and it can't be our ckeditor4 main repo slug.

So, the extra origin should be added only for PR's that come from other forks. This PR also requires changes in the travis.yml on the main repo. ~I'm going to prepare them.~ Please, take a look at: https://github.com/ckeditor/ckeditor4/pull/4972

Closes #19

sculpt0r commented:
I added an argument parser, so we can use naming parameters. Since there is a lot of arguments - it will be easier to maintain them from the command line and inside the code. Since they are mostly dependent on the CI - it is hard to hold them inside the configuration `json` file.

jacekbogdanski reviewed (changes requested):
I have a feeling that we cannot introduce such breaking change even if we can't come up quickly with anyone who would use this repository. We can assume that there are developers which are integrating CKEditor 4 CI with their own tooling, so it's risky to just break the whole bender runner command line. Also, it's hard to test these changes right now.

I propose to:
* create `development` branch for development as we are already using `master` in "production". Let's freeze `master` for some time (e.g. 2 releases of CKE4)
* start versioning bender runner (I've already created v0.1.0 for `master`)
* use versioned version for CKE4 instead of `master` branch in https://github.com/ckeditor/ckeditor4/pull/4972 so we can see if it's working properly
* create PR in CKE4 which addressing the issue in #19, so we can see everything works as expected
* publish bender runner on NPM to simplify the whole process (we may skip this part for now)

sculpt0r commented:
Ready for review:
- added `.editorconfig` since trimming the last empty line by VSC was a little bit frustrating
- added logs with exit codes - because the spawned process may exit properly, but with error code and data outputted on standard error output
- Bumped package version to `1.0.0` since we are introducing breaking change with naming parameters
- yargs parser was downgraded to be compatible with Node 10

https://github.com/ckeditor/ckeditor4/pull/4972 PR contains changes that should be synced with this PR. The Travis cfg temporarily uses this repo on this branch (until we create the first release). Also, there is a dummy change in one plugin to verify if diff query is generating properly. This change should be reverted before the actual merge.

jacekbogdanski commented on package.json:
```diff
@@ -1,22 +1,23 @@
+		"test": "mocha"
+	},
+	"author": "",
+	"license": "ISC",
```
We should use the same license and author as for other public repositories, see https://github.com/ckeditor/ckeditor4-integrations-common/blob/master/package.json#L23
We should also include the license file.

jacekbogdanski reviewed (approved):
Looks good! I've downgraded the package as versioning should happen during tagging.
